src/org/armedbear/lisp/ZipCache.java

```diff
@@ -423,18 +422,79 @@ synchronized public static Archive getArchive(JarPathname jar) {
     if (innerJars.equals(NIL)) {
       return getArchiveFile(jar);
     } else {
```

Comment:
An `else` after an early `return` is an anti-pattern!

It forces a needless extra level of indentation.